### PR TITLE
Crane UI + Tekton Integration Enhancement Updates

### PR DESCRIPTION
**Pull request by djzager**
This PR is to mainline all of the changes needed to support the Crane UI + Tekton Integration as detailed in [the enhancement](https://github.com/konveyor/enhancements/tree/master/enhancements/crane-2.0/tekton-ui-integration).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,25 +1,33 @@
-FROM registry.ci.openshift.org/openshift/release:golang-1.16 as crane-bin
+FROM registry.ci.openshift.org/openshift/release:golang-1.17 as plugin-bin
+ENV GOFLAGS "-mod=mod"
+WORKDIR /go/src/github.com/konveyor/crane-plugin-openshift
+RUN git clone -b wh-csvs https://github.com/djzager/crane-plugin-openshift.git .
+RUN go build -a -o /build/OpenShiftPlugin cmd.go openshift.go
```
Comment (djzager) [inline Dockerfile:5]: Probably need to drop this :sunglasses: but we'll need to have a new release of the plugin before this PR merges.

Any thoughts @JaydipGabani?

Alternatively, I could leave it until we are ready to cut a "release" of some kind.

File: config/clustertasks/crane-apply.yaml [hunk 2]
```diff
@@ -3,22 +3,19 @@ kind: ClusterTask
...
   annotations:
-    migration.openshift.io/run-after: "crane-transform"
     description: |
       This is where a really long-form explanation of what is happening in
       crane-apply ClusterTask would go.
 spec:
   steps:
     - name: crane-apply
-      image: quay.io/konveyor/crane-runner:latest
+      image: quay.io/djzager/crane-runner:alpha
```
Comment (djzager) [inline config/clustertasks/crane-apply.yaml:12]: Need to fix this too.

File: Dockerfile [hunk 3]
```diff
@@ -1,25 +1,33 @@
...
+    tar xvzf - -C /usr/local/bin/ kustomize
+RUN kustomize build /config/default > /deploy.yaml
+
+FROM registry.redhat.io/ubi8/ubi:latest
+COPY --from=crane-bin  /build/crane /usr/local/bin/crane
+COPY --from=cli-bin    /usr/bin/oc /usr/bin/oc
+COPY --from=cli-bin    /usr/bin/kubectl /usr/bin/kubectl
+COPY --from=cli-bin    /usr/local/bin/kustomize /usr/local/bin/kustomize
+COPY --from=cli-bin    /usr/local/bin/kubectl-convert /usr/local/bin/kubectl-convert
+COPY --from=cli-bin    /deploy.yaml /deploy.yaml
```
Comment (djzager) [inline Dockerfile:27]: Want to update mtk-operator to use the yaml in the container...we'll see how that goes.

**Review by jmontleon — COMMENTED** [review]
These changes have been working for me when testing as a non-admin

**Review by jmontleon — APPROVED** [review]
VISIACK

**Pull request merged**